service/updaters/property_by_postcode_updater_v3.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import re
+from service import date_utils
+from service import es_utils
+from service.database import page_reader
+from service.updaters.base import AbstractIndexUpdater
+
+LOGGER = logging.getLogger(__name__)
+POSTCODE_REGEX = r'[A-Z]{1,2}[0-9R][0-9A-Z]? [0-9][A-Z]{2}'
+
+
+class PropertyByPostcodeUpdaterV3(AbstractIndexUpdater):
+    """elasticsearch data updater for property_by_postcode doc type in version 3"""
+
+    def get_next_source_data_page(self, page_size):
+        source_data_page = page_reader.get_next_data_page(
+            self.last_updated_title_number,
+            self.last_title_modification_date,
+            page_size,
+        )
+
+        return source_data_page
+
+    def prepare_elasticsearch_actions(self, title):
+        if title.is_deleted:
+            return self._prepare_delete_actions(title)
+        else:
+            return self._prepare_upsert_actions(title)
+
+    def get_mapping(self):
+        return {
+            'properties': {
+                'title_number': {'type': 'string', 'index': 'no'},
+                'postcode': {'type': 'string', 'index': 'not_analyzed'},
+                'street_name': {'type': 'string', 'index': 'no'},
+                'house_no': {'type': 'integer', 'index': 'no'},
```

Comment:
no :sob: now added some extra code to work around this and yet keep integer-style ordering for the house_no